internal/supervisor/ensure_kill_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
Was this intended?

Reply:
No, but it seems to get automatically added by gofmt. If I remove `//go:build linux` then save the file, it gets re-added. Occasionally I need to run `gofmt -w .` manually because VS Code doesn't seem to always format Go on save in the same way it does just about everything else. If it's not needed, thoughts on how to not get it re-added?

Reply:
Looks like it's fine.  `go:build linux` is the newer syntax.